Sources/OpenAISwift/OpenAISwift.swift

```diff
@@ -25,17 +25,54 @@ extension OpenAISwift {
         let body = Command(prompt: prompt, model: model.modelName, maxTokens: maxTokens)
```

Comment:
suggestion: here the data you could add is in your Enum, Sources/OpenAISwift/OpenAIEndpoint.swift. so your enum will handle all your request params which is the Path, method, source url, and the data.

Example: 

```
    public var body: Data? {
        switch self {
        case .send(let advisorMessage):
            return try? JSONEncoder().encode(advisorMessage)
        }
    }
```